src/Core/Framework/Adapter/Twig/Extension/PcreExtension.php

```
@@ -33,7 +33,7 @@ public function getFunctions(): array
      */
     public function pregReplace(string $subject, string $pattern, string $replacement): string|array
     {
-        $value = preg_replace($pattern, $replacement, $subject);
+        $value = @preg_replace($pattern, $replacement, $subject);
```

below is an if to check the function return, suppress does not break anything

---

I suppose it's to fix `preg_replace(): Delimiter must not be alphanumeric, backslash, or NUL` can we simply check we have a delimiter within $pattern maybe? `/` or `#`

---

the unit test expectedly throws an invalid pattern inside, to see what happens. I would not do any regex checks, it's too complicated 🙈

---

yes indeed, in this specific usage and context we just want to throw an exception once the result is false and not try to cover preg_* 👍